lib/widgets/action_sheet.dart

```diff
@@ -1602,3 +1612,49 @@ class EditButton extends MessageActionSheetMenuItemButton {
     composeBoxState.startEditInteraction(message.id);
   }
 }
+
+class DeleteButton extends MessageActionSheetMenuItemButton {
+  DeleteButton({super.key, required super.message, required super.pageContext});
```

Comment:
nit: make private?

Or if not, then give a more explicit name: e.g. DeleteMessageButton.